docs/source/en/using-diffusers/inference_with_tcd_lora.md

```diff
@@ -0,0 +1,433 @@
+
+> ***Freely Change the Detailing:*** During inference, the level of detail in the image can be simply modified by adjusing one hyper-parameter gamma. This option does not require the introduction of any additional parameters.
+
+For more technical details of TCD, please refer to [the paper](https://arxiv.org/abs/2402.19159).
+
+Trajectory consistency distillation can directly place on top of a pre-trained diffusion model as a LoRA module. Such LoRA can be identified as a versatile acceleration module applicable to different fine-tuned models or LoRAs sharing the same base model without the need for additional training.
+
+TCD-LoRAs are available for [stable-diffusion-v1-5](https://huggingface.co/runwayml/stable-diffusion-v1-5), [stable-diffusion-2-1-base](https://huggingface.co/stabilityai/stable-diffusion-2-1-base), and [stable-diffusion-xl-base-1.0](https://huggingface.co/stabilityai/stable-diffusion-xl-base-1.0). 
+
+The corresponding checkpoints can be found at [TCD-SD15](https://huggingface.co/h1t/TCD-SD15-LoRA), [TCD-SD21-base](https://huggingface.co/h1t/TCD-SD21-base-LoRA) and [TCD-SDXL](https://huggingface.co/h1t/TCD-SDXL-LoRA), separately.
+
+
+This guide shows how to perform inference with TCD-LoRAs for 
+- text-to-image
+- inpainting
+- community models
+- style LoRA
+- ControlNet
+- IP-Adapter
+- AnimateDiff
+
+TCD-LoRA can be considered an advanced method compared with [LCM-LoRA](https://latent-consistency-models.github.io/). The guide of TCD-LoRA workflow is:
+- Load the task specific pipeline and model.
+- Set the scheduler to [`TCDScheduler`].
+- Load the TCD-LoRA weights for the model.
+- Set the `num_inference_steps` between [4, 50].
+- Set `eta` from [0, 1]. Larger `eta` in [`TCDScheduler`] will lead to blurrier images.
+- Perform inference with the pipeline with the usual parameters.
+
+Let's look at how we can perform inference with TCD-LoRAs for different tasks.
+
+First, make sure you have [peft](https://github.com/huggingface/peft) installed, for better LoRA support.
+
+```bash
+pip install -U peft
+```
+
+## Text-to-image
+
+You can use the [`StableDiffusionXLPipeline`] with the scheduler: [`TCDScheduler`] and then load the TCD-LoRA. Together with the TCD-LoRA and the TCDScheduler, the pipeline enables a fast inference workflow with high quality outputs.
+
+```python
+import torch
+from diffusers import StableDiffusionXLPipeline, TCDScheduler
+
+device = "cuda"
+base_model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+tcd_lora_id = "h1t/TCD-SDXL-LoRA"
+
+pipe = StableDiffusionXLPipeline.from_pretrained(base_model_id, torch_dtype=torch.float16, variant="fp16").to(device)
+pipe.scheduler = TCDScheduler.from_config(pipe.scheduler.config)
+
+pipe.load_lora_weights(tcd_lora_id)
+pipe.fuse_lora()
+
+prompt = "Beautiful woman, bubblegum pink, lemon yellow, minty blue, futuristic, high-detail, epic composition, watercolor."
```

Comment:
Could we try to make use of non-human characters? Since it's the official documentation, I am slightly concerned about the reception of this. 

WDYT? @yiyixuxu @stevhliu

Reply:
Let's replace it with a cute cat.

Reply:
Yeah works for me. Could you generate the results for those as well?

Reply:
For [animagine-xl-3.0](https://huggingface.co/cagliostrolab/animagine-xl-3.0) and IP-Adapter, the human characters is performing better. Do we need to forcefully replace these?

Reply:
I will let @stevhliu comment further here.

Reply:
Should be ok for this example 👍